### PR TITLE
Regionality metric & modern vizlab

### PR DESCRIPTION
**Pull request by lindsayplatt**
This calculates regionality based on work we did 2 years ago and adds a rudimentary graph to the `index.html` (see below), all using the modern vizlab framework. Just run `vizlab::vizmake()` and you should see this on the front page:

image

Calculation method still needs verification + need better viz of this metric :) Both of those are planned work for other people.

Related to https://github.com/usgs-makerspace/wbeep-sandboxr/issues/14

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 info:
   id: internal-analytics
   name: Internal analytics reporting
-  date: 2017-04-03
+  date: 2018-10-15
```
Comment (wdwatkins) [inline viz.yaml]: Could bump this up

**Comment by ldecicco-USGS** [comment]
I don't remember either.....at one point we needed added the compression because we had such a small setup on Jenkins....but then we got a bigger memory footprint...I might have un-compressed them.... I think either will work now.

**Comment by lindsayplatt** [comment]
Hmm I didn't realize something I had done overwrote that. So where should I change `saveRDS`? Or should I?

File: viz.yaml (hunk 1 above)
Comment (lindsayplatt) [reply viz.yaml]: Committing change in just a few min

**Comment by wdwatkins** [comment]
It's in all the `saveRDS` calls. e.g. here: https://github.com/USGS-VIZLAB/internal-analytics/pull/228/files#diff-9edf981e79692fc872681e16d4bf7ad7L23.  It may not actually be something you changed, just got pulled along since this branch got so old?  I'd say just leave it for now, we can change it later if things get really slow .

**Comment by wdwatkins** [comment]
Go ahead and merge when you're ready

**Pull request merged**